wasm_tests/non-nullable-funcrefs.wat

```diff
@@ -0,0 +1,8 @@
+(module
+  (type $t (func (param i32) (result i32)))
+  (func $f1 (param i32) (result i32) local.get 0)
+  (func $f2 (param i32) (result i32) local.get 0)
+  (table $t 1 1 (ref null $t))
+  (elem $t (i32.const 0) (ref null $t)
+        (item (ref.func $f1))
+        (item (ref.func $f2))))
```

Comment:
Is there a test harness in the tree that use these tests in wasm_tests? I looked around and didn't see one.

Reply:
Unfortunately there wasn't, actually! I had been meaning to write one, but mostly gathered WATs here that I manually roundtripped. I'll go ahead and put something together now that tests roundtripping, and add a note pointing to fuzzing for deeper/more authoritative correctness testing.

Reply:
Added now! I took your suggestion to test idempotency after two roundtrips -- the `roundtrip` integration test does that now using this test suite.

(Also, `test3.wat` was malformed so I removed it -- goodness knows what I was attempting with it at the time...)